notebooks/without_learning.py

```diff
@@ -0,0 +1,156 @@
+# Begin Code
```

Comment:
Should this file be checked in? I thought we were planning to keep the notebooks as the master copy and only use the Python files for testing and static analysis.